aspnetcore/mvc/views/partial.md

```diff
@@ -88,13 +86,13 @@ When a partial view is instantiated, it gets a copy of the parent view's `ViewDa
 You can pass an instance of `ViewDataDictionary` to the partial view:
 
 ```cshtml
-@Html.Partial("PartialName", customViewData)
+@await Html.PartialAsync("PartialName", customViewData)
 ```
 
-You can also pass a model into a partial view. This can be the page's view model, or some portion of it, or a custom object. You can pass a model to `Partial`,`PartialAsync`, `RenderPartial`, or `RenderPartialAsync`:
+You can also pass a model into a partial view. This can be the page's view model, or some portion of it, or a custom object. You can pass a model to `PartialAsync` or `RenderPartialAsync`:
```

Comment:
In the second sentence, the pronoun "it" is difficult to grok ...

> This can be the page's view model, or some portion of it, or a custom object.

Some portion of the page's view model? ... can you clarify? ... or just remove that phrase?

If you keep something in the middle of that list, kill that middle "or" word.

Reply:
I realize this was pre-existing to your updates; but if you could do a quick fix there, that would be awesome!

Reply:
Simple update is just to kill it ...

> This can be the page's view model or a custom object.

Reply:
I just removed it.  No problem that it was existing, whatever you all want changed is fine with me.